src/components/Header/HeaderContact.js

```diff
@@ -90,12 +90,16 @@ const styles = StyleSheet.create({
   }),
   text: {
     color: COLORS.black,
-    fontFamily: fonts.inter[600],
+    fontFamily: fonts.inter[500],
     fontSize: normalizeFontSize(14),
-    fontWeight: 'bold',
     padding: 10,
     textAlign: 'center'
   },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the `text` style object seem to be aimed at changing the font family and removing the bold weight. This could potentially impact the appearance of any text using this style. Ensure that this change aligns with your design guidelines.
```diff
- fontFamily: fonts.inter[600],
- fontWeight: 'bold',
+ fontFamily: fonts.inter[500],
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->